exercises/clock/canonical-data.json

```diff
@@ -213,7 +213,12 @@
           "minute": 1,
           "add": 3500,
           "expected": "11:21"
-        },
+        }
+      ]
+    },
+    {
+      "description": "Subtract minutes",
+      "cases": [
         {
           "description": "subtract minutes",
           "property": "add",
```

Comment:
This feels a bit funny. We tell we want to subtract, but still test the property add.

Shouldn't we introduce a property `sub` then, or omit subtracting at all?